src/loki_alert_rules/high_error_rate.rule

```diff
@@ -0,0 +1,8 @@
+alert: HostHighLogErrorRate
+expr: count_over_time({job="varlogs"} |= "error" [1h]) > 100
```

Comment:
Confirm that these rules would be rendered with otelcol topology labels.
We can perhaps own the "100" threshold for otelcol's own logs, but not if this rule is going to be evaluated for all apps.